factory/glideFactoryMonitorAggregator.py

```diff
@@ -501,11 +404,13 @@ def aggregateStatus(in_downtime):
                 a_el = int(tp_el[a])
                 val_dict[f"{tp_str}{a}"] = a_el
 
-    glideFactoryMonitoring.monitoringConfig.write_rrd_multi("total/Status_Attributes", "GAUGE", updated, val_dict)
+    glideFactoryMonitoring.monitoringConfig.write_rrd_multi(
+        os.path.join("total", "Status_Attributes"), "GAUGE", updated, val_dict
+    )
 
     # Frontend total rrds across all factories
     for fe in list(status_fe["frontends"].keys()):
-        glideFactoryMonitoring.monitoringConfig.establish_dir("total/%s" % ("frontend_" + fe))
+        glideFactoryMonitoring.monitoringConfig.establish_dir(os.path.join("total", "frontend_" + fe))
```

Comment:
Would it be better to change `"frontend_" + fe` to `f"frontend_{fe}"`?

Reply:
done